clang/utils/TableGen/ClangOpenCLBuiltinEmitter.cpp

```diff
@@ -896,9 +896,8 @@ static void OCL2Qual(Sema &S, const OpenCLTypeStruct &Ty,
     if (ImageTypesMap.contains(T->getValueAsString("Name")))
       continue;
     // Check we have not seen this Type
-    if (TypesSeen.contains(T->getValueAsString("Name")))
+    if (!TypesSeen.try_emplace(T->getValueAsString("Name"), true).second)
```

Comment:
This is pretty weird, can we make it a StringSet instead? Doesn't look like the value is used.

Reply:
The value doesn't seem to be used indeed.  Let me clean that up in a subsequent patch.